src/platform/plugins/shared/newsfeed/public/lib/driver.test.ts

```diff
@@ -67,9 +67,12 @@ describe('NewsfeedApiDriver', () => {
         .toPromise();
 
       expect(fetchMock.lastUrl()).toEqual('http://newsfeed.com/8.0.0/news');
-      expect(fetchMock.lastOptions()).toEqual({
-        method: 'GET',
-      });
+      expect(fetchMock.lastOptions()).toMatchInlineSnapshot(`
+        Object {
+          "body": Promise {},
+          "method": "GET",
+        }
+      `);
```

Comment:
Alternatively, something like this works as well, if we want to avoid snapshots. However, I don't think it matters much in this case.
`expect(fetchMock.lastOptions()).toEqual(expect.objectContaining({ method: 'GET' }))`